api/routers/v1/projects.py

```diff
@@ -28,3 +30,35 @@ async def get_projects(db: Session = Depends(get_db), offset: int = 0, limit: in
 @router.get("/{project_id}", response_model=ProjectSchema)
 async def get_project(project_id: int, db: Session = Depends(get_db)):
     return db.query(Project).filter(Project.id == project_id).first()
+
+
+@router.get("/{project_id}/allocations", dependencies=[Depends(BearerToken())])
+async def get_project_allocations(
+    project_id: int,
+    start: date = date.today(),
+    end: date = date.today(),
+    current_user=Depends(get_current_user),
```

Comment:
Do we need the `current_user` here? It doesn't appear that we are using any of its properties in the service call.

And will we worry about adding permissions a bit later? Totally fine if yes - just want to make sure it wasn't missed.

Reply:
True, for now we are not using but we definitely will need for the permissions - only PMs should be able to manage the allocations

Reply:
Sounds good. We can add later as we continue to implement and refine.